chart/values.schema.json

```
@@ -1629,6 +1666,18 @@
                     "description": "Annotations to add to the triggerer pods.",
                     "type": "object",
                     "default": {}
+                },
+                "securityContext": {
+                    "description": "Global Pod security context as defined in `Pod Security Context <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritycontext-v1-core>`_. If not set, the values from `securityContext` will be used.",
```

```suggestion
                    "description": "Triggerer Pod security context as defined in `Pod Security Context <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritycontext-v1-core>`_. If not set, the values from `securityContext` will be used.",
```

Or similar, and for all the others that aren't actually global?